requirements.txt

```diff
@@ -3,3 +3,4 @@ numpy
 scipy
 six==1.13.0
 scikit-learn==0.22.1
+Pillow
```

Comment:
Would it be possible to define a specific version?

Reply:
Sure. I'll set to the current version 7.0.0